### PR TITLE
(RA) Updating the template for more configurability

### LINKED ISSUE #66: add step to setup to cd ios and run pod install
Currently, when you use the template to start a new project, you can't run it immediately. You have to `cd ios` and run `pod install`.

The goal of this task is to add this step to the setup file.

### PR DESCRIPTION
**Pull request by rynatk**
- prompt user for git repo initialization
- prompt user for ios cocoapods
- organize splashscreen setup per platform
- use `displayName` in bundler name prompt

Things to check later:
- react-native-gesture-handler is required by react-navigation
- automatic linking of react-native-gesture-handler does not work for ios
- ~the splashscreen is not funcitonal in the generated android build~

Fixes #66

### CODE REVIEW

**Comment by jsjoeio** [comment]
Verified that everything still works
image

**Comment by jsjoeio** [comment]
Appears to be working on android as well
image

**Comment by jsjoeio** [comment]
Suggested change from Chris:
image

**Comment by jsjoeio** [comment]
Using the suggested change from @cball still works when generating a new app
image

File: setup/index.js [hunk 1]
```diff
@@ -34,6 +34,18 @@ async function setup() {
...
+    type: 'confirm',
+    name: 'shouldInitializeGitRepository',
+    message: 'Would you like to initialize this project as a git repository?',
+  });
+
+  const { shouldInitializeCocoapods } = await prompt({
+    type: 'confirm',
+    name: 'shouldInitializeCocoapods',
+    message: 'Would you like to use cocoapods in the ios app?',
+  });
```
Comment (cball) [inline setup/index.js]: we can take this out, as all new apps will use cocoapods

File: setup/index.js [hunk 2]
```diff
@@ -61,11 +73,20 @@ async function setup() {
     { cwd: rootDirectory },
   );
 
-  console.log('\n🌊  Setting up splash screens...');
-  execSync('rm -rf ios/HelloWorld/Base.lproj/LaunchScreen.xib', { cwd: rootDirectory });
+  if (shouldInitializeCocoapods) {
```
Comment (cball) [inline setup/index.js]:
```suggestion
```

File: setup/index.js [hunk 3]
```diff
@@ -61,11 +73,20 @@ async function setup() {
...
+    console.log('\n☕  Installing Cocoapods...');
+    execSync('bundle exec pod install', { cwd: 'ios' });
+  }
```
Comment (cball) [inline setup/index.js]:
```suggestion
```

File: setup/index.js (hunk 1 above)
Comment (jsjoeio) [reply setup/index.js]: Good call, removed!

**Pull request merged**